src/contrib/mlir/core/ngraph_dialect/fused_ops.td

```diff
@@ -32,14 +33,895 @@ def NGSqueezeOp :
   }];
 
   let parser = [{ NGRAPH_CHECK(false, "No parser support"); return mlir::failure(); }];
+  let verifier = [{ return mlir::success(); /* TBD */ }];
+}
+
+// Unsqueeze Op
+def NGUnSqueezeOp : 
+    NG_OneResult_Op<"unsqueeze", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data, NG_TensorType:$axes)>
+{
+  let summary = "Unsqueeze Op";
+  let description = [{
+    Unsqueeze Op
+  }];
+
+  let parser = [{ NGRAPH_CHECK(false, "No parser support"); return mlir::failure(); }];
+  let verifier = [{ return mlir::success(); /* TBD */ }];
+}
+
+// Squared Difference Op
+def NGSquaredDiffOp : 
+    NG_Binary_Op<"sqrdDiff", [DeclareOpInterfaceMethods<FusedOp>]>
+{
```

Comment:
1. So, we also need to define the arguments here, right? comparing with https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/op/fused/squared_difference.hpp#L42
2. Why is this NG_Binary_Op vs NG_OneResult_Op? `y[i] = (x1[i] - x2[i])^2` that is the equation, right?

Reply:
`NG_Binary_Op` will have the results and arguments defined (lhs, rhs). Doesn't matter what the equation is, just a binary operation.